### PR TITLE
feat(IconType): eye-visibility-svg added to the IconType

### PR DESCRIPTION
**Pull request by rahul-p-rajesh**
## Description
The strikethrough eye icon has been added to the Icons module. Now, whenever the user clicks on it and the password becomes visible, this icon can be used.

### Checklist:
- [✅] My changes generate no new warnings
- [✅] I have added tests that prove my fix is effective or that my feature works
- [✅] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: projects/assets-library/src/icons/icon-library.module.ts [hunk 1]
```diff
@@ -58,6 +58,7 @@ const iconsRootPath = 'assets/icons';
         { key: IconType.External, url: `${iconsRootPath}/external.svg` },
         { key: IconType.ExternalApi, url: `${iconsRootPath}/external-api.svg` },
         { key: IconType.Eye, url: `${iconsRootPath}/eye.svg` },
+        { key: IconType.EyeVisibilityOff, url: `${iconsRootPath}/visibility_off.svg` },
```
Comment (palbizu) [inline projects/assets-library/src/icons/icon-library.module.ts:61]: please replace '_' by a '-', ex: visibility-off,svg

**Review by palbizu — APPROVED** [review]
LGTM

File: projects/assets-library/src/icons/icon-library.module.ts (hunk 1 above)
Comment (rahul-p-rajesh) [reply projects/assets-library/src/icons/icon-library.module.ts]: renamed the visibility icon to visibility-off.svg

File: projects/assets-library/src/icons/icon-type.ts [hunk 2]
```diff
@@ -61,6 +61,7 @@ export const enum IconType {
   External = 'svg:external',
   ExternalApi = 'svg:external-api',
   Eye = 'svg:eye',
+  EyeVisibilityOff = 'svg:visibility_off',
```
Comment (itssharmasandeep) [inline projects/assets-library/src/icons/icon-type.ts:64]: use hyphen (`-`) here as well instead of underscore (`_`)

**Pull request merged**